### PR TITLE
Add focus to the first field on form

### PR DESCRIPTION
**Pull request by pusztaienike**

### CODE REVIEW

**Comment by herflis** [comment]
- [ ] Add a new property (autoFocus? boolean) to   getControlForContentField method in the control mapper
- [ ] Add a list of control names to the EditView.tsx (Name, FileName, ShortText, Autocomplete, Textarea, Number)
- [ ] add the prop autoFocus to the getControlForContentField method, and set its value true for the first control whoes name exists in the list
- [ ] handle autoFocus property in the mapper to send its value to the actual controls
- [ ] Modify the listed fieldcontrols (both in the admin-ui and the react-controls package) to handle autoFocus prop

**Comment by pusztaienike** [comment]
> * [ ]  Add a new property (autoFocus? boolean) to   getControlForContentField method in the control mapper
> * [x]  Add a list of control names to the EditView.tsx (Name, FileName, ShortText, Autocomplete, Textarea, Number)
> * [ ]  add the prop autoFocus to the getControlForContentField method, and set its value true for the first control whoes name exists in the list
> * [x]  handle autoFocus property in the mapper to send its value to the actual controls
> * [x]  Modify the listed fieldcontrols (both in the admin-ui and the react-controls package) to handle autoFocus prop

Please check the current solution, we don't need to add the autoFocus property to getControlForContentField

**Review by herflis — APPROVED** [review]

**Comment by taki9** [comment]
quill (rich text editor) could support auto focus too?

File: packages/sn-controls-react/src/viewcontrols/EditView.tsx [hunk 1]
```diff
@@ -124,11 +128,16 @@ export const EditView: React.FC<EditViewProps> = (props) => {
                 fieldOnChange: handleInputChange,
                 extension: props.extension,
                 uploadFolderPath: props.uploadFolderpath,
+                autoFocus: hasInputField.includes(field.controlType.name) && !isAutofocusSet ? true : false,
               },
             )
 
             const isFullWidth = isFullWidthField(field, props.contentTypeName)
 
+            if (hasInputField.includes(field.controlType.name) && !isAutofocusSet) {
```
Comment (taki9) [inline packages/sn-controls-react/src/viewcontrols/EditView.tsx:137]:
```javascript
const autoFocus = hasInputField.includes(field.controlType.name) && !isAutofocusSet
...
autoFocus // (where props is given)
...
if (autoFocus) {
  isAutofocusSet = true
}
```

**Review by taki9 — APPROVED** [review]

**Pull request merged**